src/Core/Url/QueryParam.php

```diff
@@ -72,11 +72,15 @@ public function isRaw()
 
     /**
      * Generate the paramatere to be appended to the url
-     * @return string the parameter on this format: ``name=value``
+     * @return string the parameter on this format: ``name=value`` or ``name``
      */
     public function generate()
     {
-        return $this->getName() . '=' . $this->getValue();
+        $result = $this->getName();
+        if (!is_null($this->value)) {
+            $result .= '=' . $this->getValue();
+        }
+        return $result;
```

Comment:
soft reset, make your 2 commits, and push with force. But for this time it's ok, and the change is minor !